pkg/allocator/allocator.go

```diff
@@ -714,21 +722,27 @@ func (a *Allocator) Allocate(ctx context.Context, key AllocatorKey) (idpool.ID,
 		value, isNew, firstUse, err = a.lockedAllocate(ctx, key)
 		if err == nil {
 			a.mainCache.insert(key, value)
-			log.WithField(fieldKey, key).WithField(fieldID, value).Debug("Allocated key")
+			a.logger.Debug("Allocated key",
+				fieldKey, key,
+				fieldID, value,
```

Comment:
`fieldID` could be likely replaced in favor of `logfields.ID` (even though the latter is uppercase).